spec/requests/api/v1/people_controller_image_spec.rb

```diff
@@ -13,28 +13,47 @@
         allow(NomisClient::Image).to receive(:get).and_return(nil)
       end
 
-      it 'returns 404' do
-        get "/api/v1/people/#{person.id}/image", params: { access_token: token.token }
+      it 'returns resource not found 404' do
+        id = 'non-existent-id'
+
+        get "/api/v1/people/#{id}/images", params: { access_token: token.token }
+
         expect(response).to have_http_status(:not_found)
       end
     end
 
     context 'with an image' do
+      subject(:get_image) { get "/api/v1/people/#{person.id}/images", params: { access_token: token.token } }
+
       let!(:person) { create(:profile, :nomis_synced).person }
       let(:image_data) { File.read('spec/fixtures/Arctic_Tern.jpg') }
 
+
       before do
         allow(NomisClient::Image).to receive(:get).and_return(image_data)
-
-        get "/api/v1/people/#{person.id}/image", params: { access_token: token.token }
       end
 
       it 'returns success' do
+        get_image
+
         expect(response).to be_successful
       end
 
-      it 'returns the image' do
-        expect(response.body).to eq(image_data)
+      it 'contains the url of the image' do
+        get_image
+
+
+        expect(JSON.parse(response.body)['data']['attributes']['url']).to include person.id + '.jpg'
```

Comment:
IMHO we can just check for equality of the JSON reply - it's not too big or onerous

Reply:
that would imply to generate in full the URL, I think this test works and it is explicit